docs/pages/guides/how-expo-works.md

```diff
@@ -55,9 +55,8 @@ The following is an example of a manifest being served through Expo CLI. The fir
     "urlRandomness":"2v-w3z",
     "lanType":"ip"
   },
-  "xde":true,
   "developer":{
-    "tool":"xde"
+    "tool":"expo-cli"
```

Comment:
Ensure this works with Expo Go.

Reply:
I've tested using Expo Go on Android and iOS now. Also note that this line (`developer.tool`) is not changing (it has always been `expo-cli` when using the CLI).